nni/tools/nnictl/config_schema.py

```diff
@@ -130,7 +130,7 @@ def validate(self, data):
     Optional('maxTrialDuration'): And(Regex(r'^[1-9][0-9]*[s|m|h|d]$', error='ERROR: maxTrialDuration format is [digit]{s,m,h,d}')),
     Optional('maxTrialNum'): setNumberRange('maxTrialNum', int, 1, 99999),
     'trainingServicePlatform': setChoice(
-        'trainingServicePlatform', 'remote', 'local', 'pai', 'kubeflow', 'frameworkcontroller', 'dlts', 'aml', 'adl', 'hybrid'),
+        'trainingServicePlatform', 'remote', 'local', 'pai', 'kubeflow', 'frameworkcontroller', 'dlts', 'aml', 'adl', 'hybrid', 'dlc'),
```

Comment:
If you provide v1 configuration, please add corresponding convert in https://github.com/microsoft/nni/blob/master/nni/experiment/config/convert.py#L16.

Reply:
remove v1 support.